publication/src/main/scala/hmda/publication/reports/disclosure/DisclosureB.scala

```diff
@@ -35,13 +42,18 @@ trait DisclosureB extends DisclosureReport {
 
     val lars = larSource
       .filter(lar => lar.geography.msa != "NA")
```

Comment:
The nationwide reports need to include LARs with MSA=NA

Reply:
Good to know. Done in 944a428c26bc9bcba7e0e27245de0b1ab0f056ee.